.github/workflows/pull_request.yml

```diff
@@ -26,6 +26,29 @@ jobs:
         run: go get
       - name: Build
         run: CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+          cache: false
+      - name: Go linting
+        uses: golangci/golangci-lint-action@v4
+        with:
+          version: v1.54
```

Comment:
Is this the latest version we can use, think it is on 1.57 at the moment?

Reply:
I've changed it to 1.57, I used 1.54 since that's they had in their example for implementation into actions, 1.57 works so I've updated it 👍